GSoC_submission.md

```diff
@@ -0,0 +1,59 @@
+# GSoC Webservices 2019 Submission
+
+This page to record the progress made during Google Summer of Code program in the last three months.
+
+## About the project
+
+This project is a continuation of the implementation of webservices in Joomla. Webservices are API layer that helps third-party developers work with a Joomla website. This can significantly improve the popularity of Joomla CMS, because it will help to create very functional and complex integrations with sites made on Joomla.
```

Comment:
```suggestion
This project is a continuation of the implementation of webservices in Joomla. Webservices are the API layer that helps third-party developers work with a Joomla website. This can significantly improve the popularity of Joomla CMS, because it will help create very functional and complex integrations with sites made on Joomla.
```